src/commonMain/kotlin/org/mongodb/kbson/BsonDecimal128.kt

```diff
@@ -39,6 +39,24 @@ public class BsonDecimal128 private constructor(internal val value: Decimal128)
     override val bsonType: BsonType
         get() = BsonType.DECIMAL128
 
+    /**
```

Comment:
JavaDoc is the wrong way round. Should be "Gets the high-order bits"

Reply:
Ups. Copy-pasted too quickly there. It has been fixed.